### PR TITLE
Cucumber

### PR DESCRIPTION
**Pull request by jaffamonkey**
Ready for merging for master - pretty please.

### CODE REVIEW

File: web/sites/default/settings.local.travis.php [hunk 1]
```diff
@@ -1,7 +1,5 @@
 <?php
 
-$settings['hash_salt'] = 'THIS_IS_THE_HASH_SALT';
```
Comment (stemount) [inline web/sites/default/settings.local.travis.php]: check this

File: web/sites/default/settings.local.travis.php [hunk 2]
```diff
@@ -23,7 +21,5 @@
   $settings['container_yamls'][] = $app_root . '/' . $site_path . '/services.local.yml';
 }
 
-$settings['trusted_host_patterns'] = [];
```
Comment (stemount) [inline web/sites/default/settings.local.travis.php]: and this

File: tests/wdio.conf.js [hunk 3]
```diff
@@ -123,16 +123,13 @@ exports.config = {
...
     // },
-    reporters: ['json', 'spec', 'allure'],
+    reporters: ['json', 'spec'],
     reporterOptions: {
         json: {
             outputDir: './reports/',
             filename: 'report',
             combined: true,
         },
-        allure: {
```
Comment (stemount) [inline tests/wdio.conf.js]: prob is - but is this to change?

**Comment by stemount** [comment]
Is this supposed to still be open? Whatcha think @jaffamonkey

**Comment by stemount** [comment]
Only really need the changes of `web/sites/default/settings.local.travis.php` undoing :)

**Comment by kalpaitch** [comment]
@chris-moreton @stemount All tests are passing, the only reason this is failing is because the file permissions on Travis are not allowing temporary local caches to be generated. This is causing display errors:

error_chrome_2017-10-13t10-25-53 195z

TEMPTED TO MERGE THIS IN AND LET ALL BRANCHES FAIL UNTIL THIS IS RESOLVED... WE NEED TO GET TEST RUNNING BEFORE WE DO ANOTHER DEPLOY ANYWAY.

**Comment by stemount** [comment]
`TEMPTED TO MERGE THIS IN AND LET ALL BRANCHES FAIL UNTIL THIS IS RESOLVED... WE NEED TO GET TEST RUNNING BEFORE WE DO ANOTHER DEPLOY ANYWAY.`

> SOUNDS GOOD

**Pull request merged**